2020-03-Autumn/1_pointer_techniques/QuestionOne/driver.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#import <vector>
+using namespace std;
+
+int removeDuplicates(std::vector<int>& nums) {
+
+    if(nums.size() == 0){
+        return 0;
+    }
+
+    //Pointer to the last, unique element in the array
+    int left = 0;
+    int right = 1;
+    int size = nums.size();
+
+    //Iterate through thr entire vector
+    while ( right < size ){
```

Comment:
Small nitpick, but I would try and make the spacing between the parentheses consistent so it looks a bit prettier.